generators/readme/src/index.ts

```diff
@@ -0,0 +1,170 @@
+import type { VFile, VFileCompatible } from 'vfile';
+import type { Node } from 'unist';
+import type { Parent, Heading, Text, Paragraph, Code, Root, HTML } from 'mdast';
+import type { Options as PrettierOptions } from 'prettier';
+import type {
+  CommandFlags,
+  GeneratorReadmeOptions,
+  PackgeJsonInfo,
+} from './types';
```

Comment:
can be moved to the bottom of the imports since it is relative import

Reply:
No, this is an `import type` which I prefer to keep at the top, before the normal imports.